### DISCUSSION TITLE
Linkerd Behaviour of Failing Pods

### DISCUSSION THREAD

**Discussion by luqmansen** [linkerd/linkerd2#6645] [Q&A]
Hi, I have a question regarding the Linkerd behaviour when a pod is failing, 

Suppose one request coming to a pod, but before it is reaching the pod, the pod is crashed, does Linkerd provide a request autoroute for that particular request to another pod, or let that request failed?

I read that Linkerd has a fail-fast mechanism that will buffer a request for around 3 seconds before it started to fail the request, does the scenario that I mention count?

Thank you

**Comment by adleong** [comment]
Yes, Linkerd supports reties https://linkerd.io/2.10/features/retries-and-timeouts/

**Reply by luqmansen** [reply]
Thank you for the answer
